Formula/lgogdownloader.rb

```diff
@@ -0,0 +1,46 @@
+class Lgogdownloader < Formula
+  desc "Open source downloader for GOG.com games"
+  homepage "https://sites.google.com/site/gogdownloader"
+  url "https://sites.google.com/site/gogdownloader/lgogdownloader-3.2.tar.gz"
+  sha256 "9e60af50c18533884ca317778b291152d5f04d93dfd3dc2579f6f965538b2031"
+
+  depends_on "cmake" => :build
+  depends_on "help2man" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "boost"
+  depends_on "htmlcxx"
+  depends_on "jsoncpp"
+  depends_on "liboauth"
+  depends_on "rhash"
+  depends_on "tinyxml2"
+
+  def install
+    mkdir "build" do
+      system "cmake", "..", "-DCMAKE_INSTALL_PREFIX=#{prefix}",
+             "-DCMAKE_BUILD_TYPE=Release"
+      system "make", "install"
+    end
+  end
+
+  test do
+    require "pty"
+
+    assert_equal "LGOGDownloader #{version}",
+      pipe_output("#{bin}/lgogdownloader --version").chomp
+
+    login_cmdline = %W[#{bin}/lgogdownloader --list --retries 1]
+    test_credentials = {
+      :username => "test@example.com\n",
+      :password => "secret\n",
+    }
+    ENV["XDG_CONFIG_HOME"] = mktemp(:retain => true) { pwd }
+    ohai login_cmdline.join(" ")
```

Comment:
Was this for debugging?

Reply:
It’s deliberate; its purpose is to show which of the two tests are executing, and in the event of failure, to determine which test has failed.

In the first test, the `pipe_output` method logs its command line as well. It just does it internally.

Reply:
Does the fork/exec not work here?

Reply:
Nope, it doesn’t because `lgogdownloader` requires a TTY for the username/password input. None of the existing shell helpers seem to support that.

Reply:
I don't think we need this ohai. If it's unclear where it's failing add another assert in between

Reply:
We have two tests here, which are being executed in sequence:

```
$ brew test lgogdownloader
Testing lgogdownloader
==> /usr/local/Cellar/lgogdownloader/3.2/bin/lgogdownloader --version
==> /usr/local/Cellar/lgogdownloader/3.2/bin/lgogdownloader --list --retries 1
```

The first `ohai` happens implicitly inside `pipe_output`.
The second `ohai` is the explicit one.
If I leave it out, the test suggests that the `--version` part is failing. That’s confusing to the end user. Hence, the manual `ohai`.

How would an additional assert in between help with that?